src/main/java/DilemmaDetector/Consequences/ConsequenceContainer.java

```diff
@@ -29,9 +29,16 @@ public void saveConsequencesToOntology() {
 
         for (String decisionName: materialConsequencesByDecisionMap.keySet()){
             Set<Map.Entry<String, Double>> entries = materialConsequencesByDecisionMap.get(decisionName).entrySet();
+            Double sum = 0.0;
+            Material_consequence material_consequence = factory.createMaterial_consequence(ObjectNamer.getName("material_consequence"));
             for(Map.Entry<String, Double> entityConsequence : entries){
-                saveMaterialConsequence(decisionName, entityConsequence.getKey(), entityConsequence.getValue());
+                sum += entityConsequence.getValue();
+                Entity entity = factory.getEntity(entityConsequence.getKey());
+                material_consequence.addMaterial_consequence_to(entity);
             }
+            Decision decision = factory.getDecision(decisionName);
+            decision.addHas_consequence(material_consequence);
+            material_consequence.addHas_material_value(sum.floatValue());
         }
     }
 
```

Comment:
To teraz mam jedną konsekwencję dla decyzji, i wiele połączeń do encji za pomocą material_consequence_to

![image](https://user-images.githubusercontent.com/47008811/97698022-2905b580-1aa8-11eb-8d8f-e19afd61c312.png)

Tu wartość kolizji wyszła duża bo dałem dużą wartość ciężarówce :)